hrrecruitment/tests/phpunit/CRM/HRRecruitment/BAO/HRVacancyTest.php

```diff
@@ -49,8 +37,11 @@ protected static function _populateDB($perClass = FALSE, &$object = NULL) {
    */
   function testCreateGet() {
 
-    $permissionIndividual1Id = Contact::createIndividual();
-    $permissionIndividual2Id = Contact::createIndividual();
+    $params1 = array('first_name' => 'micky', 'last_name' => 'mouse');
+    $permissionIndividual1Id = $this->createContact($params1);
```

Comment:
The thing is: if there's already something else that does the same job, why duplicate the code?

But, on a second thought, after the discussion about the API and after I saw that the Contact::createIndividual() method was just removed in a core PR (https://github.com/civicrm/civicrm-core/pull/8643/files#diff-808104459263d190f0d695fd6e29464eL30) and probably it won't be available in the future, I think is fine to keep the helper method you just created.

About the milliseconds, that is REALLY important in tests. You're supposed to be able to run them several times during development and every millisecond counts in this situation